### PR TITLE
Tests for logout controller

### PR DESCRIPTION
**Pull request by rj13371**
### What this PR does (required):
made ping test for logout controller to see if it returns a 200 status code

### Any information needed to test this feature (required):

Made Readme in server/README.md

### CODE REVIEW

**Review by moffatethan — APPROVED** [review]
Good work!

**Review by moffatethan — CHANGES_REQUESTED** [review]
Forgot to mention, please base the branch properly. 

[Read how to change the base here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

File: server/test/ping.spec.js [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+    request(url, function (error, response, body) {
+      expect(response.statusCode).to.equal(200);
+    });
   });
+
 });
+
+describe("ping for logout post request", function () {
+
+  var url = "http://localhost:3001/auth/logout/"
```
Comment (moffatethan) [inline server/test/ping.spec.js]: I would prefer a `const` being used here just for convention in the overall project.

File: server/test/ping.spec.js [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-          .eql("Server is running. Message received: hello");
-        done();
-      });
+var expect = require('chai').expect;
+var request = require('request');
+
+
+describe("profile GET route", function () {
+
+  var url = "http://localhost:3001/profiles/profiles"
```
Comment (moffatethan) [inline server/test/ping.spec.js]: I would prefer a const being used here just for convention in the overall project.

File: server/test/ping.spec.js [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
-      .send({ message: "hello" })
-      .end((err, res) => {
-        res.should.have.status(200);
-        res.body.should.have
-          .property("response")
-          .eql("Server is running. Message received: hello");
-        done();
-      });
+var expect = require('chai').expect;
+var request = require('request');
```
Comment (moffatethan) [inline server/test/ping.spec.js]: `const` can be used here too. Thanks!

**Review by moffatethan — CHANGES_REQUESTED** [review]

**Comment by rj13371** [comment]
I changed the base to profile_controllers here, which is what I worked on before working on this branch. Is this supposed to be based on what I was working on before?

**Comment by moffatethan** [comment]
> I changed the base to profile_controllers here, which is what I worked on before working on this branch. Is this supposed to be based on what I was working on before?

Yes, exactly 👍🏻 💯

**Review by moffatethan — APPROVED** [review]

**Pull request merged**